WordPress/Classes/Networking/ReaderTopicServiceRemote.m

```diff
@@ -2,7 +2,9 @@
 #import "WordPressComApi.h"
 #import "RemoteReaderTopic.h"
 #import "ReaderTopic.h"
+#import "RemoteReaderSiteInfo.h"
 
+NSString * const WordPressComReaderEndpointURL = @"https://public-api.wordpress.com/rest/v1.2/";
```

Comment:
I don't think so. The 1.2 endpoint has some changes that could affect other parts of the app, even other parts of the reader. Right now, we need it just when fetching posts.